helios/management/commands/import_election_polls_ballots.py

```diff
@@ -16,34 +16,35 @@ class Command(BaseCommand):
     @transaction.atomic
     def handle(self, *args, **options):
         election = Election.objects.get(uuid=args[0])
-        data = yaml.load(open(args[1]))
-        for poll_data in data:
-            if not poll_data.get('uuid', None):
-                poll = Poll(election=election)
-            else:
-                poll = election.polls.get(uuid=poll_data.get('uuid'))
-            poll.name = poll_data.get('name')
-
-            questions = poll_data.get('questions')
-            questions_data = []
-            for qdata in questions:
-                question = qdata.get('question')
-                answers = qdata.get('answers')
-                _min, _max = qdata.get('min', 1), qdata.get('max', 1)
-                q = {
-                    'choice_type': 'choice',
-                    'question': question,
-                    'max_answers': _max,
-                    'min_answers': _min,
-                    'answers': answers
-                }
-                for i, ans in enumerate(answers):
-                    q['answer_%d' % i] = ans
-                questions_data.append(q)
-            poll.questions_data = questions_data
-            poll.update_answers()
-            poll.save()
-            poll_data['uuid'] = str(poll.uuid)
-
-        write = open(args[1], 'w')
-        yaml.dump(data, write, allow_unicode=True, default_flow_style=False)
+        with open(args[1]) as f:
```

Comment:
`f` is used only in the next line, so it's better to do it like this:

```
with open(args[1]) as f:
    data = yaml.load(f)

for poll_data in data:
    ...
```

This way, you close `f` immediately after reading the file.